cwms-data-api/src/test/resources/cwms/cda/data/dto/stream_location.json

```diff
@@ -0,0 +1,22 @@
+{
+  "stream-location-id": {
+    "office-id": "SPK",
+    "name": "StreamLoc123"
+  },
+  "stream-node": {
+    "stream-id": {
+      "office-id": "SPK",
+      "name": "ImOnThisStream"
+    },
+    "bank": "R",
+    "station": 678.9,
+    "station-unit": "ft"
+  },
+  "published-station": 123.45,
```

Comment:
unit for station would be inferred by the stream-node?

Reply:
correct